### ISSUE TITLE
Mines do not work well on rates; Mines are enforced; Optimize this

### ISSUE THREAD

**Issue by DeltaEpsilon7787** [etternagame/etterna#326]
tl;dr: Mines should probably stay where they are relative to other notes and not be affected by rates.

Now that mines are enforced, a certain other issue pops up occasionally, namely: mines are affected by rates.

Due to things like debounce time, polling rate and such of player's keyboard, mines should always stay a certain period of time away from the closest note. Most charters seem to have figured out a safe range of up to ~90 ms for mine dodge charts. Go any closer and artificial difficulty is created, making mine-intensive charts retarded on higher rates.

As such, something needs to be done about it.

**Comment by MinaciousGrace** [comment]
mine's aren't enforced; they're required to be on for leaderboard rankings

the point of that being that everyone ranked on the leaderboard is playing under the set of threats

there are issues with mines sure but this would just create conflicts with notes being placed into max mine range on rates for some files and if you deal with that by negating those mines then you've fundamentally changed how the threats work, thus invalidating the level playing field concept

you dont have to play with mines
you dont have to play files with mines on rates

pretty sure you could make nomines versions of every chart in existence ask them to be ranked and nobody would care
